.env.example.local

```diff
@@ -19,10 +19,13 @@ UAA_LOGOUT_PATH=/logout.do
 # CF API
 #   Used to connect to the Cloud Foundry API. CF_API_URL should always end
 #   with /v3 regardless of the environment.
-#   The CF_API_TOKEN can be populated with `cf oauth-token` or by running
+#   Locally, use CF_USER_ID to get CAPI info related to the current logged in user.
+#   In a deployed environment, this user id comes from the auth token.
+#   Locally, the CF_API_TOKEN can be populated with `cf oauth-token` or by running
 #   npm run dev-cf
 #
 CF_API_URL=https://api.dev.us-gov-west-1.aws-us-gov.cloud.gov/v3
+CF_USER_ID=your-cf-user-guid
```

Comment:
Could we add and inline comment or step in the README to describe where to retrieve your `CF_USER_ID`?

Reply:
Certainly—updated with additional instructions.